vars/MPLModule.groovy

```diff
@@ -70,6 +70,13 @@ def call(String name = env.STAGE_NAME, Map cfg = null) {
     Helper.runModule(module_src, module_path, [CFG: Helper.flatten(cfg)])
   }
   catch( ex ) {
+    
+    if (ex instanceof org.jenkinsci.plugins.workflow.steps.FlowInterruptedException) {
```

Comment:
Why don't use specialized catch for that instead of if?

Reply:
Didn't think of it (I am not an expert in groovy). You are right.